theme_v2_json_structure.py

```diff
@@ -0,0 +1,351 @@
+        "social_media_image": ' url',
+        "meta_description": {
+            "ar": "المعهد العقاري",
+            "en": "SREI excellence"
+        },
+    },
+
+    "footer": {
+        "sections": [  #--  API key: footer_sections
+             {
+                 "key": "option_1",
+                 "link_title": {
+                     "ar": "",
+                     "en": ""
+                 },
+                 "page": "courses",  # or home, about_us, pages_custom_page_1, ... or https://
+                 #-- Display one of the list of pages,
+                 "new_tab": True,
+             },
+             {
+                 "key": "option_2",
+                 "link_title": {
+                     "ar": "",
+                     "en": ""
+                 },
+                 "page": "about_us",  # or home, about_us, pages_custom_page_1, ... or https://
+                 #-- Display one of the list of pages,
+                 "new_tab": True,
+             },
+             {
+                 "key": "option_3",
+                 "link_title": {
+                     "ar": "",
+                     "en": ""
+                 },
+                 "page": "pages_custom_page_1",  # or home, about_us, pages_custom_page_1, ... or https://
+                 #-- Display one of the list of pages,
+                 "new_tab": True,
+             },
+        ],
+        "social_media_links": [  #--  API key: footer_social_media_links
+            {
+                "provider": "facebook",
+                "value": "url"
+            },
+            {
+                "provider": "x",
+                "value": "url"
+            }
+        ]
+    },
+
+    "header": {
+        "sections": [  #--  API key: header_sections
+            {
+                "key": "option_1",
+                "link_title": {
+                    "ar": "",
+                    "en": ""
+                },
+                "page": "courses",  # or home, about_us, pages_custom_page_1, ... or https://
+                #-- Display one of the list of pages,
+                "new_tab": True,
+            },
+            {
+                "key": "option_2",
+                "link_title": {
+                    "ar": "",
+                    "en": ""
+                },
+                "page": "about_us",  # or home, about_us, pages_custom_page_1, ... or https://
+                #-- Display one of the list of pages,
+                "new_tab": True,
+            },
+            {
+                "key": "option_3",
+                "link_title": {
+                    "ar": "",
+                    "en": ""
+                },
+                "page": "pages_custom_page_1",  # or home, about_us, pages_custom_page_1, ... or https://
+                #-- Display one of the list of pages,
+                "new_tab": True,
+            },
+        ],
+        "logo": {  #--  API key: header_logo
+            "ar": "url",
+            "en": "url"
+        },
```

Comment:
the one in the original settings
```suggestion
        "LOGO_URL": "url"
```